.github/renovate.json

```diff
@@ -0,0 +1,28 @@
+{
+  "extends": [
+    "config:base"
+  ],
+  "packageRules": [
+    {
+      "matchPaths": [".github/workflows"],
+      "groupName": "Github Workflows",
+      "schedule": ["on the first day of the week"]
+    },
+    {
+      "matchPackagePatterns": ["eslint"],
+      "groupName": "eslint",
+      "schedule": ["on the first day of the month"]
+    },
+    {
+      "matchPackagePatterns": ["jest"],
+      "groupName": "jest",
+      "schedule": ["on the first day of the month"]
+    },
+    {
+      "matchPackagePatterns": ["^express"],
+      "matchPackageNames": ["cookie-parser", "helmet"],
+      "groupName": "express",
+      "schedule": ["on the first day of the month"]
+    }
```

Comment:
For the rest of our dependencies, can we create another group like so:
```
    {
      "matchPackagePatterns": [
        "*"
      ],
      "groupName": "all dependencies",
      "groupSlug": "all",
       "schedule": ["on the first day of the month"]
    }
```
We can keep other groups for more sensitive dependencies

Reply:
I dont think we can, there are quite a few of the other dependancies that require manual steps and cannot be bumped together.

Reply:
All good then, worth a shot, ill approve